extensions/blocks/opentable/opentable.php

```diff
@@ -55,18 +32,7 @@ function register_block() {
  * is loaded.
  */
 function set_availability() {
-	if ( is_available() ) {
-		\Jetpack_Gutenberg::set_extension_available( BLOCK_NAME );
-	} else {
-		\Jetpack_Gutenberg::set_extension_unavailable(
-			BLOCK_NAME,
-			'missing_plan',
-			array(
-				'required_feature' => 'opentable',
-				'required_plan'    => 'premium-plan',
-			)
-		);
-	}
+	\Jetpack_Gutenberg::set_availability_for_plan( FEATURE_NAME );
 }
 add_action( 'jetpack_register_gutenberg_extensions', 'Jetpack\OpenTable_Block\set_availability' );
```

Comment:
Should we make this part of every block, rather than having to call it separately?

Reply:
Yes, I was thinking about that but as this is primarily regarding gating features to plans I thought that it should be left as a check that can be used for those blocks. There's some other availability logic in some of the blocks, so this could be used in conjunction with that.

On the other hand, if the majority of blocks will need a plan check, then we could register this hook as the blocks are registered, and then remove it on the others. Another option might be to pass another argument to `jetpack_register_block` that could indicate the check is needed.

That last option might be the cleanest. What do you think about that?

Reply:
I think lets keep it explicit right now and revisit if we find we're copy/pasting this code a lot.

Reply:
Sounds good!